acceptance/lib/puppet/acceptance/common_utils.rb

```diff
@@ -56,12 +55,7 @@ def crontab_entries_of(host, username)
         crontab_contents.lines.map(&:chomp).reject { |line| line =~ /^#/ }
       end
 
-      # Due to OSX 10.14 Mojave new security feature called “Full Disk Access”
-      # that limits the operations that a OSX user can do,
-      # we can not manage users properly using puppet.
-      # We will use an already existing user('osx') to run the cron tests.
-      def cron_user(agent)
-        return 'osx' if agent[:platform] =~ /osx-10.14/
+      def cron_user
```

Comment:
Does this need to be copied to the `cron_core` module?

Reply:
checked and is not needed, this was introduced by a previous try of mine to fix OS X tests. cron_core is also using `tstuser` for tests